src/parse/errors.rs

```diff
@@ -310,8 +310,9 @@ pub enum ErrorKind {
     /// ```
     UnexpectedMultipleUsage,
 
-    /// Occurs when the user provides a value containing invalid UTF-8 for an argument and
-    /// [`AppSettings::StrictUtf8`] is set.
+    /// Occurs when the user provides a value containing invalid UTF-8 for an argument.
```

Comment:
The external subcommand value containing utf8 also throws this error. Do we want to rephrase this or create a new error kind?

Reply:
I think they are similar enough that people would want one error for both cases.  I'll look over the wording